askbot/deps/group_messaging/models.py

```diff
@@ -77,6 +77,7 @@ class LastVisitTime(models.Model):
 
     class Meta:
         unique_together = ('user', 'message')
+        app_label = 'askbot'
```

Comment:
The database was creating models as group_messaging_CLASS rather than askbot_CLASS. Django was searching for the latter table, so I added this meta_data to ensure the prefix was askbot instead